elements/pl-graph/pl-graph.py

```diff
@@ -0,0 +1,105 @@
+import prairielearn as pl
```

Comment:
If possible, in the future, could you `git mv old_file new_file`? History was lost on this file :/  (Though, glancing at the other diff, I'm not sure that's a bad thing.)

Reply:
Git actually doesn’t differentiate between `git mv` and the pair of `git rm` + `git add`. It only stores file contents, and does “rename detection” for display. See https://stackoverflow.com/questions/7938582/how-does-git-detect-similar-files-for-its-rename-detection